dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/ModuleMergeData.java

```diff
@@ -120,9 +122,9 @@ BeanMergeData createBeanMergeDataFromXML(String name, String className) {
     }
 
     private ModuleAnnotations getModuleAnnotations() {
-        if (moduleAnno == null) {
+        if ( moduleAnno == null ) {
             try {
-                moduleAnno = container.adapt(ModuleAnnotations.class);
+            	moduleAnno = AnnotationsBetaHelper.getModuleAnnotations(container);
```

Comment:
I see you changed this in multiple places.  Was it not possible just to have the adapt code call the AnnotationsBetaHelper so you only have to change the adapt code instead of all the places that call adapt?

Reply:
It would.  I noticed this same improvement a bit late in the implementation and didn't have an opportunity to adjust the implementation.

Reply:
Are you going to change it as part of this pull request?

Reply:
I wasn't planning to. The necessary updates go beyond the modifications to the adapt call.  In addition to the adapt update, there would need to be updates to these:

open-liberty/dev/com.ibm.ws.app.manager.war/src/com/ibm/ws/app/manager/ear/internal/EARDeployedAppInfo.java
See: EARDeployedAppInfo.hasAnnotations

open-liberty/dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/archive/liberty/CDIArchiveImpl.java
See: CDIArchiveImpl.getAnnotatedClasses

WS-CD-Open/dev/com.ibm.ws.eba.jpa.container.annotations/src/com/ibm/ws/eba/jpa/annotation/scanning/BlueprintAnnotations.java
WS-CD-Open/dev/com.ibm.ws.eba.jpa.container.annotations/src/com/ibm/ws/eba/jpa/annotation/scanning/BlueprintAnnotationsImpl.java
WS-CD-Open/dev/com.ibm.ws.eba.jpa.container.annotations/src/com/ibm/ws/eba/jpa/annotation/scanning/BlueprintAnnotationsAdapterImpl.java
WS-CD-Open/dev/com.ibm.ws.eba.jpa.container.annotations/src/com/ibm/ws/eba/jpa/annotation/scanning/JpAnnotationSource.java

Reply:
I've created this issue to address this update:
https://github.com/OpenLiberty/open-liberty/issues/9215